app/views/catalog/_show_default.html.erb

```diff
@@ -20,11 +16,8 @@
           <% search = heading['search2'] %>
           <% else search = heading['search1']  %>
           <% end %>
-          <%# show heading popover for everything bigger than smartphone %>
-          <a href="#" role="button" tabindex="0" data-trigger="focus" data-auth='<%=  search %>' data-poload='/browse/info?authq=<%=  search.gsub("&", "%26") %>&amp;browse_type=Author&amp;headingtype=<%=  type %>&amp;bib=<%=document.id%>' id="info" class="info-button d-none d-sm-inline-block btn btn-sm btn-outline-secondary" aria-label="<%=  search %>" onclick="javascript:_paq.push(['trackEvent', 'itemView', 'author-info-popup']);">Author info »</a>
 
-          <%# link directly to heading on smartphone %>
-          <a href="/browse/info?authq=<%=  search.gsub("&", "%26") %>&amp;browse_type=Author&amp;headingtype=<%=  type %>" role="button" class="info-button d-inline-block d-sm-none btn btn-sm btn-outline-secondary" aria-label="<%=  search %>"  onclick="javascript:_paq.push(['trackEvent', 'itemView', 'author-info-mobile']);">Author info »</a>
+          <a href="/browse/info?authq=<%=  search.gsub("&", "%26") %>&amp;browse_type=Author&amp;headingtype=<%=  type %>" role="button" class="info-button btn btn-sm btn-outline-secondary" aria-label="<%=  search %>"  onclick="javascript:_paq.push(['trackEvent', 'itemView', 'author-info']);">Author info »</a>
```

Comment:
Looks like we're missing a `bib` parameter here which is needed for returning back to item after clicking through to the Author browse page. Will push up a commit for this in a second and test on int.

Reply:
Good catch! Would not have thought of that. (And fingers crossed tests pass with this PR.)